logging_metrics_performance/hawkular_metrics/scripts/README_metricsChecker.md

```diff
@@ -0,0 +1,39 @@
+## metrics_checker README
+
+### Purpose 
+The **metrics_checker.py** script is a tool for verifying hawkular metrics in OpenShift.  It can be used to verify that metrics data is present in Cassandra for an arbitrary amount of time in the past.
+
+
+### Running the tool
+
+
+1. Create a user with cluster-admin privileges which can be used to query metrics.  Log that user in and get the login token (redhat is an example user):
+
+```
+oc adm policy add-cluster-role-to-user cluster-admin redhat
+oc login -u redhat -p redhat
+oc whoami -t
+```
+
+Save the value of the token
+
+2. Create the projects and pods
+
+3. Run metrics_checker.py - this example uses metrics buckets of 2 minutes duration and checks for data in the last 5 buckets.  See below for all flags:
+
+**python metrics_checker.py -B <token> -p svt -H <hawkular route or internal IP> -s "-10mn" -d "120s" -b 5 -i 30**
+
+This translates to "Check projects starting with svt.  Check past 5 2 minute buckets.  Check 5 pods at a time and repeat every 30 seconds"
+
+
+### Complete metrics_checker.py flags 
+
+```python ocp_logtest.py <optional-arguments>```
```

Comment:
typo in python script name, needs to be:  python metrics_checker.py <optional-arguments>